src/System/Random/Stateful.hs

```diff
@@ -105,10 +105,11 @@ module System.Random.Stateful
   ) where
 
 import Control.DeepSeq
+import Control.Monad (replicateM)
 import Control.Monad.IO.Class
 import Control.Monad.ST
 import GHC.Conc.Sync (STM, TVar, newTVar, newTVarIO, readTVar, writeTVar)
-import Control.Monad.State.Strict
+import Control.Monad.State.Strict (MonadState, state)
```

Comment:
I can't see why these changes are needed, can you revert them?

Reply:
They are needed. [Changelog](https://hackage.haskell.org/package/mtl-2.3/candidate/changelog) says "Remove re-exports of `Control.Monad`, `Control.Monad.Fix` and `Data.Monoid` modules".

Reply:
Gotcha. It does make sense now. Thank you.